Homework5/Task3/Classes/VacationInformationTools.cs

```diff
@@ -0,0 +1,114 @@
+﻿using System.Linq;
+using Task3.Records;
+
+namespace Task3.Classes
+{
+ public static class VacationInformationTools
+ {
+  public static double GetAverageVacationDuration(List<EmployeeVacationInformation> list)
```

Comment:
change input from List<T> to IEnumerable<T> , it is a common practice so method can work with any collection.
https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/guidelines-for-collections#collection-parameters